UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/BodyPartTrauma.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace HealthV2
+{
+	public class BodyPartTrauma : MonoBehaviour
```

Comment:
maybe should inherit from body part functionality?

Reply:
no

Reply:
why is this a separate class from TraumaLogic?

Reply:
BodyPartTrauma is the root that identfies traumas a body part can have.
TraumaLogic is in the name, its logic a trauma has.